### DISCUSSION TITLE
Export cleanup profile request

### DISCUSSION THREAD

**Discussion by the-afoh** [bleachbit/bleachbit#1720] [Ideas]
Hi, I'm a Linux user, I like BleachBit full functionality, but often when you're dealing with multiple systems it can be tedious to select each and every one of the things you'd like to cleanup and deselect those you don't, so I'd like to suggest an export cleanup profile feature so that you can use some file (Idk, xml or something, I'm not used to the inner workings), to restore your cleanup preferences in any installation or upgrade of a system you'd like to clean. Thanks!

**Comment by az0** [comment]
How about copying the file `~/.config/bleachbit/bleachbit.ini` between computers?
